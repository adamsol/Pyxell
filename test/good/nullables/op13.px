
func f(x: Int): [Int]? def
    print x
    x *= 2
    return x >= 0 ? [x] * x : null

print f(100)?.length, f(-1)?.length
print f(-3)?[0], f(3)?[0]
print f(4)?[f(3)!.length:], f(-4)?[:f(0)![0]]
