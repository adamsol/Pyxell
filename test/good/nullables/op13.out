100
-1
200 null
-3
3
null 6
4
3
-4
[8, 8] null
